Content.Server/Maps/GameMapManager.cs

```diff
@@ -111,7 +111,9 @@ public bool CheckMapExists(string gameMap)
 
     private bool IsMapEligible(GameMapPrototype map)
     {
-        return map.MaxPlayers >= _playerManager.PlayerCount && map.MinPlayers <= _playerManager.PlayerCount;
+        return map.MaxPlayers >= _playerManager.PlayerCount &&
+               map.MinPlayers <= _playerManager.PlayerCount &&
```

Comment:
Shouldn't the player levels be made in to a `GameMapCondition`?

Reply:
Opted against it as EVERY MAP has player min/max, so it'd just be extra typing for no gain.

Reply:
dunno how universal the player count restriction will be, especially if servers have a steady general population level, and most of their maps cater thereto

anyway :thumbsup:

Reply:
i agree with efr that playercount should p be a condition and not hardbaked into the gamemapprototype, but i wont force you to change it in this pr.